ascii-dev/mt-react
# #160921759 Setup ESlint

#### What does this PR do?
To configure ESlint linter for the application.
#### Description of Task to be completed?
- create eslint config file
#### How should this be manually tested?
After cloning the repo and running `npm install`
You should see the `.eslintrc` file in the project
#### What are the relevant pivotal tracker stories?
[#160921759](https://www.pivotaltracker.com/story/show/160921759)
#### Questions
N/A
